qutip/core/cy/coefficient.pyx

```diff
@@ -499,11 +501,11 @@ cdef class InterCoefficient(Coefficient):
         return out
 
     @classmethod
-    def from_PPoly(cls, ppoly):
+    def from_PPoly(cls, ppoly, **_):
```

Comment:
I am wondering if we should keep the **_. This can lead to confusing behavior were some arguments get ignored.
```
def f(a):
    pass

f(1, **{}) # this does not raise error
f(1, **{"m" : 1})  # I think we should make this raise an error.
```

Reply:
The `**_` is not for the users defined functions, but for the function that build the `Coefficient` object.
The `coefficient` function has the `args`, `order`, `tlist`, etc. that it need to pass down, function based coefficients need the `args`, the string coefficients use the `order` and `tlist`. 
Since `coefficient` no longer have special cases for each type, we don't filter the input anymore.